config/locales/placements/en/models/hosting_environment.yml

```diff
@@ -0,0 +1,7 @@
+en:
```

Comment:
I think they will look the same for both in the beginning but according to the assumption on the ticket they might look different later

`Track & Pay and School Placements may move out of BETA at different times. We’ll likely need a way of configuring these independently.`

Reply:
I think it makes sense for them to be separate because they aren't necessarily on the same time line -- one might stop being a 'beta' service before the other.

Reply:
In that article, it also suggests to move to rack for running the system specs when you don't have javascript and use selenium when you have.

This might speed up our specs a bit considering we are quite light on javascript